### PR TITLE
Make app service plan sku size and tier parameters and add documentation

### PR DESCRIPTION
**Pull request by dayland**

### CODE REVIEW

File: docs/knownissues.md [hunk 1]
```diff
@@ -57,7 +58,10 @@ Turn off the option to require membership for the Azure Active Directory Enterpr
...
+Occasionally you will hit a 429 return code in the FileFormRecSubmissionPDF which indicates that you need to retry your submission later, or an internal error returned by AI Document Intelligence in the FileFormRecPollingPDF function, which indicates the service has hit internal capacity issues. Both of these situations will occur under heavy load, but the accelerator is designed to back off and retry at a later time, up to a maximum set of retries, which is configurable. 
+
+### Solution
+These values surface as configuration settings in the Azure function and can be revised there, or they can be updated at deployment in function.bicep, or they can be updated in the file local.settings.json which is used when debugging a function in VS Code. These values are as follows...
```
Comment (georearl) [inline docs/knownissues.md:64]: it mentions bicep

File: docs/knownissues.md [hunk 2]
```diff
@@ -9,7 +9,8 @@ Error: This subscription cannot create CognitiveServices until you agree to Resp
 
 ```
 
-**Solution** : Manually create a "Azure AI services" in your Azure Subscription and Accept "Responsible AI Notice"
+### Solution
+Manually create a "Azure AI services" in your Azure Subscription and Accept "Responsible AI Notice"
```
Comment (georearl) [inline docs/knownissues.md:13]: an "Azure AI services"

File: docs/knownissues.md (hunk 1 above)
Comment (georearl) [reply docs/knownissues.md:64]: also add they can just resubmit through the upload status page

Comment (dayland) [reply docs/knownissues.md:64]: This is old and just needs updated. I did not edit this other than just to add a header.

Comment (dayland) [reply docs/knownissues.md:64]: I put a new ticket on the board to update old Known Issue items after TF updates

**Pull request merged**